subtitles.go

```diff
@@ -230,15 +282,20 @@ type StyleAttributes struct {
 	WebVTTVertical       string
 	WebVTTViewportAnchor string
 	WebVTTWidth          string
+	WebVTTColor          string
 }
 
 func (sa *StyleAttributes) propagateSSAAttributes() {}
 
-func (sa *StyleAttributes) propagateSTLAttributes() {}
+func (sa *StyleAttributes) propagateSTLAttributes() {
+	sa.WebVTTLine = sa.WebVTTLineFromSTL()
+	sa.WebVTTPosition = sa.WebVTTPositionFromSTL()
```

Comment:
Check if `STLVerticalPosition` is not `nil` before calling `WebVTTPosition` instead:

```go
if sa.STLVerticalPosition != nil {
    sa.WebVTTPosition = sa.STLVerticalPosition.WebVTTPosition
}
```

Reply:
Done